src/pipeline_v3/cli/management.py

```
@@ -991,10 +1003,36 @@
 
     async def handle_search(self, args):
         """Handle search operations."""
+        from utils.security import InputSanitizer
```

Pylint (3.3.7)

[error] Unable to import 'utils.security'

(E0401)